openshift/cirras-underwriting-jasper-deployment.yaml

```diff
@@ -68,22 +84,16 @@ spec:
               memory: #{JASPER_MEMORY_LIMIT}#
           image: >-
             docker.io/bitnami/jasperreports-archived:latest
+          command:
+            ['sh', '-c']
+          args:
+            ['. /vault/secrets/env && catalina.sh run']
```

Comment:
I don't see a docker file in cirras-underwriting-reports folder

Reply:
There isn't a Dockerfile for it. The deployment loads the image specified in `spec.template.spec.containers.image`, which in this case is pointing at DockerHub. We'd have to find documentation on how to set or override the ENTRYPOINT and/or CMD for each Image. Not sure if it's worthwhile since we won't be using Jasper Server anymore.

Reply:
That's what I was thinking. Not sure if it's worth the effort.

Reply:
Then the question is, what state do we leave this in?